src/main/java/no/unit/nva/cristin/projects/FetchCristinProjects.java

```diff
@@ -93,26 +89,11 @@ private String getValidLanguage(RequestInfo requestInfo) throws BadRequestExcept
     }
 
     private ProjectsWrapper createProjectsWrapper(String language, String title) {
-        ProjectsWrapper wrapper = new ProjectsWrapper();
-        wrapper.hits = createProjectPresentationList(language, title);
-        wrapper.numberOfResults = wrapper.hits.size();
-
-        return wrapper;
-    }
-
-    private List<ProjectPresentation> createProjectPresentationList(String language, String title) {
         Map<String, String> cristinQueryParameters = createCristinQueryParameters(title, language);
-        List<CristinProject> projects = attempt(() ->
-            cristinApiClient.queryAndEnrichProjects(cristinQueryParameters, language)).orElseThrow();
-
-        return transformProjectsToProjectPresentations(language, projects);
-    }
 
-    private List<ProjectPresentation> transformProjectsToProjectPresentations(String language,
-                                                                              List<CristinProject> projects) {
-        return projects.stream()
-            .map(project -> presentationConverter.asProjectPresentation(project, language))
-            .collect(Collectors.toList());
+        return attempt(() ->
+            cristinApiClient.createProjectsWrapperFromQuery(cristinQueryParameters, language))
```

Comment:
rename `createProjectsWrapperFromQuery` to describe what you are actually returning. Example: 
fetchProjectsWithTitleInSpecifiedLanguageAndSatisifyingParameterCriteria